pom.xml

```diff
@@ -170,14 +170,14 @@
             <version>master-SNAPSHOT</version>
         </dependency>
     </dependencies>
-    
+
     <repositories>
-      <repository>
-        <id>jitpack.io</id>
-        <url>https://jitpack.io</url>
-      </repository>
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
```

Comment:
Why do we have this repository in pom file? Shouldn't it be defined in the user m2 setting file.

Reply:
Well, I havent added it.... but can remove..?

Reply:
So it need to be there...